프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.js

```diff
@@ -0,0 +1,6 @@
+function solution(price, money, count) {
+    const amountPerUse = Array(count).fill().map((_, idx) => (idx + 1) * price);
+    const amountPrice = amountPerUse.reduce((acc, cur) => acc + cur, 0);
```

Comment:
```suggestion
    const totalCost = price * count * (count + 1) / 2;
```
이런식으로 토탈을 계산하는 것도 좋을것 같아요 !